### ISSUE TITLE
"[error] address: more than one address in list!" when From: has multiple addresses

### ISSUE THREAD

**Issue by dkg** [astroidmail/astroid#619]
[the message format spec](https://tools.ietf.org/html/rfc5322#section-3.6.2) says that `From` can have multiple addresses, but when astroid encounters such a message, it sends this to stderr:

    [error] address: more than one address in list!

Astroid should handle this situation more gracefully.

**Comment by gauteh** [comment]
Yes, seems like we should just handle addresslists like for the to field?

ons. 16. jan. 2019 kl. 00:00 skrev dkg <notifications@github.com>:

**Comment by dkg** [comment]
yep, i think that's correct.  thanks for looking into it!
